hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacitySchedulerNewQueueAutoCreation.java

```diff
@@ -311,6 +319,7 @@ public void testAutoCreateQueueShouldFailWhenNonParentQueue()
   public void testAutoCreateQueueWhenSiblingsNotInWeightMode()
       throws Exception {
     startScheduler();
+    csConf.setLegacyQueueModeEnabled(true);
```

Comment:
Can you please elaborate / comment why this flag is set to true for this testcase in specific?

Reply:
Fixed, it should have been like:

```
    // If the new queue mode is used it's allowed to
    // create a new dynamic queue when the sibling is
    // not in weight mode
    assumeThat(csConf.isLegacyQueueMode(), is(true));
```

Reply:
thanks.